### PR TITLE
fix: avoid emit update:checked twice

### PR DESCRIPTION
**Pull request by zkwolf**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.
#3838 

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: components/vc-checkbox/src/Checkbox.jsx [hunk 1]
```diff
@@ -89,7 +89,6 @@ export default defineComponent({
       if ('checked' in props) {
         this.$refs.input.checked = props.checked;
       }
-      this.__emit('update:checked', eventObj);
```
Comment (tangjinzhou) [inline components/vc-checkbox/src/Checkbox.jsx:92]: radioGroup checkGroup 会无法触发 update:checked 这个事件 了  全局搜索下 VcCheckbox  看下 group 的逻辑

**Review by tangjinzhou — CHANGES_REQUESTED** [review]

File: components/vc-checkbox/src/Checkbox.jsx (hunk 1 above)
Comment (zkwolf) [reply components/vc-checkbox/src/Checkbox.jsx:92]: ok，我去看看

Comment (zkwolf) [reply components/vc-checkbox/src/Checkbox.jsx:92]: 这里没影响啊，https://github.com/vueComponent/ant-design-vue/blob/next/components/checkbox/Checkbox.tsx#L104 ，vcCheckbox emit change,然后通过context调用group的事件， group自己处理update:checked

Comment (tangjinzhou) [reply components/vc-checkbox/src/Checkbox.jsx:92]: ok  我看错了  看成了 group也是 checked 事件了

**Review by tangjinzhou — APPROVED** [review]

**Pull request merged**